framework/aster-frame/Cargo.toml

```diff
@@ -45,6 +45,3 @@ iced-x86 = { version = "1.21.0", default-features = false, features = [ "no_std"
 
 [features]
 intel_tdx = ["dep:tdx-guest", "dep:iced-x86"]
-# To actively recycle page table nodes while the `VmSpace` is alive, this saves
-# memory but may lead to the page table free-reuse-then-read problem.
-page_table_recycle = []
```

Comment:
@junyang-zh Let's start with the first commit. I have removed the `page_table_recycle` feature and associated code, what do you think? If we don't enable it by default and no one really benefits from it, I think it's better to remove it to reduce the maintenance burden; otherwise this feature needs to be tested in CI, which I don't think we currently do.

Also, I'd like to note that even without this feature, we can still overwrite a huge page in a PTE that points to a page table, resulting in the page table being dropped. So in fact, we are still deallocating page table nodes while the `VmSpace` is alive. Any ideas?

Reply:
I am OK regarding the removal of this feature and implementation. Leave it as a TODO.